src/twisted/conch/test/test_knownhosts.py

```diff
@@ -53,9 +53,14 @@
     b'343Hd2QHiIE0KPZJEgCynKeWoKz8v6eTSK8n4rBnaqWdp8MnGZK1WGy05MguXbyCDuTC8AmJXQ'
     b'==')
 
+ecdsaSampleEncodedKey = (
```

Comment:
Just for future reference, we should try to generate keys for tests rather than having a bunch of hard-coded ones hanging around.  This is the sort of thing automated security scanners pick up and complain about.  There's a case to be made that they're technically wrong for complaining, but they're also the sort of thing that people blindly copy/paste from tests or examples into production systems, so in practice I think they're right.

This is just repeating a pretty well-established pattern in this file, so I wouldn't blame you for doing it here, but it's something to keep in mind for the future.